cli.bats

```diff
@@ -37,14 +37,14 @@ teardown() {
     [[ "$output" == *"SUBCOMMANDS"* ]]
 }
 
-@test "succeeds: --chain=morden account new [empty options]" {
-    run $EMERALD_CLI --chain=morden account new <<< $'foo\n'
+@test "succeeds: --chain=etc-test account new [empty options]" {
```

Comment:
it's it morden?

Reply:
Yes. ETH morden chain id is 2, ETC morden - 62